vm-gcp-go/go.mod

```diff
@@ -0,0 +1,65 @@
+module vm-gcp-go
+
+go 1.17
```

Comment:
Levi recently recommended to me on one of my templates to increment this to Go 1.18.

Reply:
Ah nice, thanks. Will do.